### PR TITLE
CWR: encode files with ISO-88591-1

### PR DESCRIPTION
**Pull request by orenyodfat**
- Set pythonencoding to ISO-88591-1
- Update git submodule  :
         -fast read stdin.
         -add cwr grammar flexibility. 
         -support COM record type.
- Update readme

### CODE REVIEW

File: cwr/parser.go [hunk 1]
```diff
@@ -24,13 +24,17 @@ import (
...
+	"os"
 	"os/exec"
 )
 
 // ParseCWRFile parse a give cwr file and returns an array of registeredWorks
 func ParseCWRFile(cwrFileReader io.Reader, CWRDataApiPath string) (registeredWorks []*RegisteredWork, err error) {
 	//phase 1 : Transform cwr formatted file to cwr-json file using CWR-DataApi python script.
 	cmd := exec.Command("python3", CWRDataApiPath+"/cwr2json.py")
+	cmd.Env = append(os.Environ(),
+		"PYTHONIOENCODING=ISO-8859-1",
```
Comment (lmars) [inline cwr/parser.go]: We should add a comment explaining why we need to use an explicit encoding.

Comment (orenyodfat) [reply cwr/parser.go]: Add a comment at e32a4a1

File: cwr/parser.go [hunk 2]
```diff
@@ -24,13 +24,19 @@ import (
...
 	"io"
+	"os"
 	"os/exec"
 )
 
 // ParseCWRFile parse a give cwr file and returns an array of registeredWorks
 func ParseCWRFile(cwrFileReader io.Reader, CWRDataApiPath string) (registeredWorks []*RegisteredWork, err error) {
 	//phase 1 : Transform cwr formatted file to cwr-json file using CWR-DataApi python script.
 	cmd := exec.Command("python3", CWRDataApiPath+"/cwr2json.py")
+	//Use explicit encoding because it seems like some of the CWR files are ISO-8859-1 encoded
```
Comment (lmars) [inline cwr/parser.go]: If only _some_ of the files are ISO-8859-1 encoded then don't we need to do some form of detection?

Comment (orenyodfat) [reply cwr/parser.go]: We might need to do that. 
So far we do not have such a file (UTF-8 with a special chars included)  for a reference. 
Currently, we can parse all of the CWR sample files we got using that.
The problematic files are files which include non standard ascii chars (>127) ...when decoding that with latin1 (ISO-8859-1) all is ok.
The non problematic files just include chars in the normal ascii range (<127) so decoding that with latin1 (ISO-8859-1) does not make a difference.

Comment (lmars) [reply cwr/parser.go]: Ok let's revisit once we have tested this on more files.

**Comment by orenyodfat** [comment]
Let's wait with merging it (or just close it) as #22  is standing on the door.

**Comment by lmars** [comment]
Let's just close this PR then.

**Pull request closed without merging**